templates/CRM/Custom/Form/CustomData.tpl

```diff
@@ -9,11 +9,12 @@
 *}
 {* Custom Data form*}
 {foreach from=$groupTree item=cd_edit key=group_id name=custom_sets}
+    {* Note this `if` looks like it's needlessly assigning a var but it's also used in the included file Edit/CustomData.tpl *}
     {if $cd_edit.is_multiple and $multiRecordDisplay eq 'single'}
-      {assign var="isSingleRecordEdit" value=TRUE}
+      {assign var="isSingleRecordEdit" value=true}
```

Comment:
Because that's what the PR is about. Capital TRUE is not technically a valid value. It will get compiled as string `'TRUE'`, which happens to work out because when the var is used later it coerces to the same as boolean `true`, but `TRUE` without quotes is technically a syntax error, which you can see in smarty strict/secure mode.

Reply:
![Fascinating](https://media.giphy.com/media/sBl8Fowq0ErFC/giphy.gif)

Reply:
🖖